### PR TITLE
Fixes error messages based on design + externalizes them to be edited…

### PR DESCRIPTION
**Pull request by ajainarayanan**
… outside actual implementation

### CODE REVIEW

File: cdap-ui/app/services/constants.js [hunk 1]
```diff
@@ -31,5 +31,32 @@ angular.module(PKG.name + '.services')
...
+
+    'en': {
+      hydrator: {
+        appLabel: 'Hydrator Pipeline',
+        studio: {
+          oneSinkError: 'Please add at least one sink to your pipeline.',
+          sourceRequiredFieldsError: 'Please provide required fields for Source.',
+          sinkRequiredFieldsError: 'Please provide required fields for Sink.',
+          transformRequiredFieldsError: 'Please provide required fields for Transform.',
+          oneSourceError: 'Pipelines can only have one source. Please remove any additional sources.',
```
Comment (astewes) [inline cdap-ui/app/services/constants.js]: We should pick either "source" or "Source" (same for Transform, Sink) and use it consistently throughout these messages.

**Comment by astewes** [comment]
One minor comment otherwise LGTM

File: cdap-ui/app/services/constants.js (hunk 1 above)
Comment (awholegunch) [inline cdap-ui/app/services/constants.js]: I'd go with lowercase, and use "the..":
```
sourceRequiredFieldsError: 'Please provide all required fields for the source.',
sinkRequiredFieldsError: 'Please provide all required fields for the sink.',
transformRequiredFieldsError: 'Please provide all required fields for the transform.',

File: cdap-ui/app/services/constants.js [hunk 2]
```diff
@@ -31,5 +31,32 @@ angular.module(PKG.name + '.services')
...
+          nameValidationError: 'Pipeline names can only contain letters of the alphabet, numbers, and \'_\' Please remove other character types.',
+          sinkBranchNodeError: 'Please connect multiple sinks to the same node.',
+          branchError: 'Please remove branched connections.',
+          unconnectedNodesError: 'Please connect all nodes.',
+          circularConnectionError: 'Please remove the circular connection in this pipeline.',
+          endSinkError: 'Please end the pipeline connections in a sink.',
+          parallelConnectionError: 'Please remove parallel connections in this pipeline.'
+        },
+        wizard: {
+          welcomeMessage1: 'Hydrator makes it easy to prepare data so you',
```
Comment (awholegunch) [inline cdap-ui/app/services/constants.js]: Does this need a trailing space, or are these two not concatenated?

File: cdap-ui/app/services/constants.js [hunk 3]
```diff
@@ -31,5 +31,32 @@ angular.module(PKG.name + '.services')
...
+        appLabel: 'Hydrator Pipeline',
+        studio: {
+          oneSinkError: 'Please add at least one sink to your pipeline.',
+          sourceRequiredFieldsError: 'Please provide required fields for Source.',
+          sinkRequiredFieldsError: 'Please provide required fields for Sink.',
+          transformRequiredFieldsError: 'Please provide required fields for Transform.',
+          oneSourceError: 'Pipelines can only have one source. Please remove any additional sources.',
+          noSourceError: 'Please add a source to your pipeline',
+          nameError: 'Please name your pipeline.',
+          nameValidationError: 'Pipeline names can only contain letters of the alphabet, numbers, and \'_\' Please remove other character types.',
```
Comment (awholegunch) [inline cdap-ui/app/services/constants.js]: Pipeline names can only contain alphanumeric (\'a-z A-Z 0-9\') characters and underscores ( \'_\'). Please remove other characters.

File: cdap-ui/app/services/constants.js (hunk 2 above)
Comment (ajainarayanan) [inline cdap-ui/app/services/constants.js]: welcomeMessage1 and 2 are concatenated in UI.

**Comment by awholegunch** [comment]
LGTM

**Pull request merged**